pkg/integrations/v4/emitter/emitter_test.go

```diff
@@ -515,30 +516,31 @@ func TestEmitV3_WithTags(t *testing.T) {
 		},
 	}
 
-	integrationJsonOutput := fmt.Sprintf(integrationJsonOutput, "")
-	ma := mockAgent()
+	integrationJSONOutput := fmt.Sprintf(integrationJsonOutput, "")
+	agentContext := mockAgent()
 
 	extraLabels := data.Map{}
 	var entityRewrite []data.EntityRewrite
 	mockDME := &mockDmEmitter{}
 	mockDME.On("Send", mock.Anything)
 
-	em := &VersionAwareEmitter{
-		aCtx:        ma,
+	emtr := &VersionAwareEmitter{
+		aCtx:        agentContext,
 		ffRetriever: feature_flags.NewManager(map[string]bool{fflag.FlagProtocolV4: true}),
 		dmEmitter:   mockDME,
 	}
 
-	err := em.Emit(definition, extraLabels, entityRewrite, []byte(integrationJsonOutput))
+	err := emtr.Emit(definition, extraLabels, entityRewrite, []byte(integrationJSONOutput))
 	require.NoError(t, err)
 
-	for c := range ma.Calls {
-		called := ma.Calls[c]
+	for c := range agentContext.Calls {
```

Comment:
not really an issue, but since we rename this, I think in this particular case the variable name could be closer to the Implementation type name.

Reply:
👍  but I'll also add a @cristianciutea  linter rule to .golangcgi.yml :P 
```
linters:
  enable-all: true
  disable:
    - scopelint
    - cristianciutea
  ```

Reply:
Done in https://github.com/newrelic/infrastructure-agent/pull/1394/commits/e05a1494ea1fc9fcf319afa9dde28c68a91e4cc1 ;-)